src/main/java/org/owasp/esapi/logging/java/JavaLogFactory.java

```diff
@@ -43,10 +43,15 @@
 
 /**
  * LogFactory implementation which creates JAVA supporting Loggers.
- *
- * This implementation requires that a file named 'esapi-java-logging.properties' exists on the classpath.
- * <br>
- * A default file implementation is available in the configuration jar on GitHub under the 'Releases'
+ * <br><br>
+ * Options for customizing this configuration (in recommended order)
+ * <ol>
+ * <li>Consider using the <i>SLF4JLogFactory</i> with a java-logging implementation.</li>
+ * <li>Configure java LogManager environment properties as defined by the <i>java.util.logging.LogManager</i> API</li>
```

Comment:
I don't see any mention of "environment variables" (which generally Java tries to avoid) in the Javadoc for [java.util.logging.LogManager](https://docs.oracle.com/javase/8/docs/api/java/util/logging/LogManager.html). I only say the 2 system properties which you see to account (which are not specifically mentioned by name here, but probably should be in case others are added in the future) for in the list item below.

Reply:
from the LogManager API linked:
```
The LogManager defines two optional system properties that allow control over the initial configuration:

"java.util.logging.config.class"
"java.util.logging.config.file"
```

I will update line 50 to refer to them as "system properties" to match the language in the class' javadoc

Reply:
This change has been applied.   Still looking at addressing the other content around error handling and visibility of LogManager configuration.